docs/collections.md

```diff
@@ -1,26 +1,51 @@
 # Collections
 
-Collection is an entity which contains documents. 
+Collection is an entity which contains documents.
+
+## Collection properties
+
+General collection looks like:
+
+    {
+        "_id": "example-collection",
+        "documents_permissions": {
+            "createable": true,
+            "removeable": true,
+            "updateable": true
+        },
+        "indexes": [],
+        "is_system": false,
+        "name": "Example Collection",
+        "repr_property": "",
+        "schema": {
+            "type": "object",
+            "additionalProperties": true
+        }
+    }
+
+
+Property     | Type          | Description
+------------ | ------------- | ------------
+_id                   | string | id of a collection
+documents_permissions | object | object which contains documents access permissions
```

Comment:
Don't write `object` in description. It's already described it type. Write something like `Document access permissions`